project/src/components/offers/offers.jsx

```diff
@@ -1,20 +1,25 @@
-import React from 'react';
+import {React, useState} from 'react';
 import PropTypes from 'prop-types';
 import Card from '../card/card';
 import offerProps from '../offers/offer.props';
 
-function Offers(props) {
-  const { offers } = props;
+function Offers({offers, listHoverHandler}) {
+  const [state, setState] = useState({});
+  const cardHoverHandler = (offer) => {
+    setState(offer);
+    listHoverHandler(state);
+  };
```

Comment:
```suggestion
  const cardHoverHandler = (offer) => {
    listHoverHandler(state);
  };
```
Локальные колбеки названы через handle